src/objective-c/tests/InteropTests/InteropTests.m

```diff
@@ -563,6 +563,47 @@ - (void)testLargeUnaryRPC {
   [self waitForExpectationsWithTimeout:TEST_TIMEOUT handler:nil];
 }
 
+- (void)testUnaryResponseHandler {
+  XCTAssertNotNil([[self class] host]);
+  // The test does not work on a remote server since it does not echo a trailer
+  if ([[self class] isRemoteTest]) return;
+  __weak XCTestExpectation *expectComplete = [self expectationWithDescription:@"received complete"];
+
+  RMTSimpleRequest *request = [RMTSimpleRequest message];
+  request.responseType = RMTPayloadType_Compressable;
+  request.responseSize = 314159;
+  request.payload.body = [NSMutableData dataWithLength:271828];
+
+  GRPCMutableCallOptions *options = [[GRPCMutableCallOptions alloc] init];
+  // For backwards compatibility
+  options.transportType = [[self class] transportType];
```

Comment:
No need to set transportType because we're setting options.transport instead.

Reply:
In case a fixture does not have a `transport`, i.e. `transport == nullptr`, this will be useful.